iocage_lib/ioc_start.py

```diff
@@ -564,20 +565,21 @@ def __start_jail__(self):
                 )
 
             if not poststart_success:
+
+                iocage_lib.ioc_stop.IOCStop(
+                    self.uuid, self.path, force=True, silent=True
+                )
+
                 iocage_lib.ioc_common.logit({
                     'level': 'EXCEPTION',
-                    'message': '  + Executing exec_poststart FAILED\n'
-                    f'ERROR:\n{poststop_output}\n\nJail '
-                    f'failed to start'
+                    'message': '  + Running exec_poststart FAILED\n'
```

Comment:
Should still be executing, as that is what we're doing. It may point a user to the fact that their script is not executable as well via language